voxblox_ros/src/tsdf_server.cc

```diff
@@ -157,11 +179,11 @@ void TsdfServer::insertPointcloud(
     // Convert the PCL pointcloud into our awesome format.
     // TODO(helenol): improve...
     // Horrible hack fix to fix color parsing colors in PCL.
-    for (size_t d = 0; d < pointcloud_msg->fields.size(); ++d) {
+    /*for (size_t d = 0; d < pointcloud_msg->fields.size(); ++d) {
```

Comment:
I actually forgot about this, I had an ugly hack and was going to go back and make it cleaner.

The issue is that I am currently using a boost::bind to pass the freespace flag to the pointcloud callback. However the bind function seems to only work with a callback that has constptr arguments, which prevents us from modifying the incoming message as is done here.